build_uefi.py

```diff
@@ -212,10 +231,11 @@ def ci_copy_fd_after_single_device_building(this_target):
     ci_upload_dir = os.path.join(build_output_path, "ci_artifacts", this_target.device)
 
     # File paths, fd and img
-    input_fd_path = os.path.join(build_output_path, this_target.buildtype + "_CLANGPDB", "FV", this_target.platform.upper() + "_EFI.fd")
-    input_img_path = os.path.join(build_output_path, this_target.device + ".img")
-    output_fd_path = os.path.join(ci_upload_dir, this_target.platform.upper() + "_EFI_" + secureboot_suffix + ".fd")
-    output_img_path = os.path.join(ci_upload_dir, this_target.device + "_" + secureboot_suffix + ".img")
+    input_fd_path = os.path.join(build_output_path, this_target.buildtype + "_CLANGPDB", "FV",
+                                 this_target.platform.upper() + "_EFI.fd")
+    input_img_path = os.path.join(build_output_path, f"{this_target.device}.img")
+    output_fd_path = os.path.join(ci_upload_dir, f"{this_target.platform.upper}_EFI_{secureboot_suffix}.fd")
```

Comment:
Please fix this. This is the reason for the CI uploading the device. In the above input_fd_path, you retained "this_target.platform.upper()", but here in output_fd_path, you replaced it with "f"{this_target.platform.upper}", which caused a naming error. Normally, this part should remain the same.

For example：
this_target.platform.upper() =  SM8250
f"{this_target.platform.upper} = <built-in method upper of str object at 0x7fd7727beb30>